GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -211,9 +211,9 @@ - (instancetype)init {
   return self;
 }
 
+#if !TARGET_OS_WATCH
```

Comment:
I'm not sure that disabling this API is the right thing to do. Perhaps we can just implement the watchOS specific functionality in here?